core/src/main/scala/org/ensime/config/richconfig.scala

```diff
@@ -32,9 +34,18 @@ package object richconfig {
         name.startsWith("scala-library") && name.endsWith(".jar")
       }
 
+    def findProject(f: FileObject)(implicit vfs: EnsimeVFS) = {
```

Comment:
why didn't you just convert the `FileObject` into a `Path` and call the existing method?